src/port1.0/portutil.tcl

```diff
@@ -3260,7 +3260,14 @@ proc check_supported_archs {} {
                     lappend unsupported $arch
                 }
             }
-            ui_error "$subport cannot be installed for the configured universal_archs '$universal_archs' because the arch(s) '$unsupported' are not supported."
+            if {$unsupported ne "" && ${os.major} == 18} {
```

Comment:
This should check that `$unsupported` is only i386.

Reply:
Sure I'll change this to `i386`